pkg/util/kruise-configuration/types.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2022 The Kruise Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kruise_configuration
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	SidecarSetPatchPodMetadataWhiteListKey = "SidecarSet_PatchPodMetadata_WriteList"
```

Comment:
`SidecarSet_PatchPodMetadata_WhiteList`, must be my typo in https://github.com/openkruise/kruise/pull/992#discussion_r927357825 ...